src/Common/tests/System/Collections/ICollection.NonGeneric.Tests.cs

```diff
@@ -208,7 +208,7 @@ public void ICollection_NonGeneric_CopyTo_TwoDimensionArray_ThrowsException(int
             }
         }
 
-        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNonZeroLowerBoundArraySupported))]
+        [ConditionalTheory("System.PlatformDetection, CoreFx.Private.TestUtilities!" + nameof(PlatformDetection.IsNonZeroLowerBoundArraySupported))]
```

Comment:
Using object will not work either as that cannot be serialized into the metadata. Even it we could there isn't any nice way to get a Member/PropertyInfo in an expression there isn't a methodof or propertyof.